prow/knative/cluster/200-cpu-limitrange.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
 spec:
   limits:
     - default:
-        cpu: 4000m
+        cpu: 8000m
```

Comment:
why 8?
"16" would allow all usage

Reply:
Also, as JR points out, the recommendation is to have no CPU limit; if we think deleting that gets us that, we should do that. If not, "16" should be fine?

Reply:
When I did the test, setting it at 16 somehow failed to have a pod scheduled. Removing limit is ok too, but I'm not sure how delete works with `kubectl apply`, aka, it may be no op if we simply delete the cpu limit

Reply:
If you're scheduling to a 16 CPU node, not all of that CPU is allocatable - some of it will be set aside for node components.

We ran into this with kind jobs attempting to schedule "the whole node" on an 8 CPU node build cluster.  Scheduling at 7300m/7400m often resulted in no nodes being available.  Scheduling for 7 worked.

Reply:
Hmm, let's try deleting and see if it works? We can add it back with a different value if absolutely necessary

Reply:
@spiffxp sure you are mixing request and limit?

Reply:
I just did a test on my toy cluster, it does get updated when I remove the line of cpu limit, so removing cpu limit now